README.md

```diff
@@ -1,15 +1,31 @@
-**This is being worked on again.**
-# shopdeck
-![contributors](https://img.shields.io/github/contributors/LetsShop3DS/shopdeck) ![followers](https://img.shields.io/github/followers/LetsShop3DS) ![stars](https://img.shields.io/github/stars/LetsShop3DS/shopdeck)
-
-**this project does not cover the Wii U eshop, thanks**
-
-The official source code for the replacement server!
-# Can I use this for piracy?
-No. We do NOT support piracy.
-# In what language is this written in?
-Shopdeck is written in modern Python 3.10, using 2 web frameworks: Django & Flask.
-# Is this legal?
-Yes, it is! We use no code from Nintendo servers and the base tickets we use have no data about legit titles.
-# How to setup?
-Check out the Wiki/Development Setup for that.
+# Shopdeck: Your Nintendo 3DS eShop Replacement
+
+Note That This Is Still In W.i.P ⚠️
+
+[![contributors](https://img.shields.io/github/contributors/LetsShop3DS/shopdeck)](https://github.com/LetsShop3DS/shopdeck/graphs/contributors) [![followers](https://img.shields.io/github/followers/LetsShop3DS)](https://github.com/LetsShop3DS) [![stars](https://img.shields.io/github/stars/LetsShop3DS/shopdeck)](https://github.com/LetsShop3DS/shopdeck)
+
+## Overview
+
+Welcome to Shopdeck, the official replacement server for the Nintendo 3DS eShop. Offering an alternative source for accessing digital content, Shopdeck aims to provide a seamless experience for 3DS users. Please note that this project does not cover the Wii U eShop for now.
+
+## Overview
```

Comment:
We could remove Overview tbh, this is more related to your fork than the upstream repo